src/UKHO.S100PermitService.Common/Clients/HoldingsApiClient.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System.Text;
+
+namespace UKHO.S100PermitService.Common.Clients
+{
+    public class HoldingsApiClient : IHoldingsApiClient
+    {
+        private readonly HttpClient _httpClient;
+
+        public HoldingsApiClient(IHttpClientFactory httpClientFactory)
+        {
+            _httpClient = httpClientFactory.CreateClient();
+            _httpClient.Timeout = TimeSpan.FromMinutes(Convert.ToDouble(5));
```

Comment:
This should be read from appsettings.json, not have a magic value

Reply:
Resolved